src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -39,7 +39,7 @@ const ProfilePicture = ({
             {isAnon ? (
               <>
                 <View style={[styles.anonStyle(size), {backgroundColor: anonBackgroundColor}]}>
-                  <Text style={{fontSize: size / 1.5, textAlign: 'center', alignSelf: 'center'}}>
+                  <Text style={{fontSize: size / 2, textAlign: 'center', alignSelf: 'center'}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The font size of the anonymous emoji code has been reduced from `size / 1.5` to `size / 2`. This might make the emoji appear smaller than before, especially on larger screens. If this change is intentional and has been verified across different screen sizes, then it's fine. Otherwise, you might want to reconsider this change.
```diff
- <Text style={{fontSize: size / 1.5, textAlign: 'center', alignSelf: 'center'}}>
+ <Text style={{fontSize: size / 2, textAlign: 'center', alignSelf: 'center'}}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->